src/LanguageContext.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import * as RNLocalize from 'react-native-localize';
+import {useState, useEffect} from 'react';
+import {Language, lobot, DEFAULT_LANGUAGE} from './utils/language';
```

Comment:
I think maybe also co-locate these utils in same translation module.

Reply:
:+1: